test/symlink.js

```diff
@@ -704,62 +706,78 @@ describe('symlink stream', function() {
   it('emits an end event', function(done) {
     var symlinkStream = vfs.symlink(outputBase);
 
-    symlinkStream.on('end', done);
+    var ended = false;
+    function end() {
+      ended = true;
+    }
+
+    symlinkStream.on('end', function() {
+      expect(ended).toBeTruthy();
+      done();
+    });
```

Comment:
I think this is the wrong order, we want to se the flag in the event and handle the assertion in the callback to pipeline

Reply:
I embeded `console.log` into `end` and the event handler of `symlinkStream:end`, and the order of calls were `end() -> symlinkStream:end`.

If this order looks strange, should I modify this part as follows?
```
  symlinkStream.on('end', done);
  function end() {}
```